index.html

```diff
@@ -52,6 +52,164 @@ <h1>I'm Arturo Alvarez.Glad to see you!</h1>
+							Tu Hogar a un Clic
+						</div>
+						<div class="canopy">
+							CANOPY
+							<ul>
+								<li>Back End Dev</li>
+								<li>2018</li>
+							</ul>
+						</div>
+						<div class="extract">
+							<p>Web page design for provide clients with fast info about the properties that are on sale</p>
+						</div>
+						<ul class="technologies">
+							<li>HTML</li>
+							<li>CSS</li>
+							<li>JavaScript</li>
+						</ul>
+						<a href="https://tuhogaraunclic.com" class="see">See project</a>
+					</div>
+				</div>
+				<div class="card">
+					<div class="image-container background-light">
+						<img src="/assets/elrincon.png" alt="El Rincon">
+					</div>
+					<div class="info">
+						<div class="info-title">
+							El Rincon
+						</div>
+						<div class="canopy">
+							CANOPY
+							<ul>
+								<li>Back End Dev</li>
+								<li>2018</li>
+							</ul>
+						</div>
+						<div class="extract">
+							<p>Web page design for the menu, orders and financial administration</p>
+						</div>
+						<ul class="technologies">
+							<li>HTML</li>
+							<li>CSS</li>
+							<li>JavaScript</li>
+						</ul>
+						<a href="https://restaurante-web-eta.vercel.app/" class="see">See project</a>
+					</div>
+				</div>
+				<div class="card">
+					<div class="image-container background-dark">
+						<img src="/assets/farmaciajesus.png" alt="Farmacia Jesus">
+					</div>
+					<div class="info">
+						<div class="info-title">
+							Farmacia Jesus
+						</div>
+						<div class="canopy">
+							CANOPY
+							<ul>
+								<li>Back End Dev</li>
+								<li>2018</li>
+							</ul>
+						</div>
+						<div class="extract">
+							<p>Drug store landing page to help clients to find the best prices for their medicine</p>
+						</div>
+						<ul class="technologies">
+							<li>HTML</li>
+							<li>CSS</li>
+							<li>JavaScript</li>
+						</ul>
+						<a href="https://farmaciajesus.onrender.com" class="see">See project</a>
+					</div>
+				</div>
+			</div>
+		</section>
+		<section class="about">
+			<div class="title">
+				<p>About Myself</p>
+			</div>
+			<div class="me position">
+					<p>
+							Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+					</p>
+			</div>
+			<div class="contact position">
+				<p>LET'S CONNECT</p>
+				<ul class="social width">
+					<li>
+						<a href="https://twitter.com/" class="a">
+							<img src="/assets/twitter.svg" alt="Twitter logo">
+						</a>
+					</li>
+					<li>
+						<a href="https://www.linkedin.com/in/arturo-alvarez-martinez-ab9b77151/" class="a">
+							<img src="/assets/Linkedin icon.svg" alt="Linkedin logo">
+						</a>
+					</li>
+					<li>
+						<a href="https://github.com/Artufelii" class="a">
+							<img src="/assets/Vector.svg" alt="Github logo">
+						</a>
```

Comment:
Good job with adding some of the social media links. But let's add all of them we see on the Figma template. As I mentioned before we need to stick to the design and code it pixel perfectly.
<img width="383" alt="Screenshot 2023-03-24 at 11 29 00 PM" src="https://user-images.githubusercontent.com/25887253/227632666-3cf8e6bc-168e-423d-b4a7-f9275722876a.png">
<img width="383" alt="Screenshot 2023-03-24 at 11 31 08 PM" src="https://user-images.githubusercontent.com/25887253/227632708-e26b8df7-800b-444e-8953-a123194c57fc.png">